docs/external-classes.json

```diff
@@ -59,23 +59,17 @@
   "name": "Google\\Auth\\",
   "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
 }, {
-  "name": "Google\\GAX\\",
-  "uri": "https://googleapis.github.io/gax-php/#Google/GAX/%s.html"
+  "name": "Google\\ApiCore\\",
```

Comment:
Yes, SG

Reply:
Actually, wait - that won't work with our current documentation setup, because the `#` symbol indicates a redirect to the latest docs version, which will not contain any GAX docs. So I'm not sure what we want to do here.

Reply:
Ah, that's right - I forgot about that. Having this as-is would be fine then, but what do you think about creating an issue to track making sure we are linking to the correctly versioned documentation?

Reply:
Created a pair of issues: https://github.com/GoogleCloudPlatform/google-cloud-php/issues/757, https://github.com/googleapis/gax-php/issues/128